src/components/SearchBar/index.tsx

```diff
@@ -73,31 +74,52 @@ export default function SearchBar({
     setSearchText(suggestion)
   }
 
+  let searchResults = <div>No results</div>
```

Comment:
can we hide the no results if the search is empty?

<img width="329" alt="image" src="https://github.com/user-attachments/assets/8aec47ef-8ab7-4778-b413-f2f08a740440">

Reply:
yup